Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Built from Node latest Alpine
+FROM node:10.0
```

Comment:
from jason's comment on my kanban:

This isn't pulling alpine version, rather you would use `node:10.0-alpine`